resources/authgear/static/authgear.css

```diff
@@ -313,9 +313,55 @@ h1, h2, h3, h4, h5, h6 {
   justify-content: space-between;
 }
 
+.errors-wrapper {
+  width: 100%;
+  min-height: 32px;
+}
+
 .errors {
-  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  background-color: var(--color-error-shaded-2);
+}
+
+.errors.display-none {
+  display: none;
+}
```

Comment:
This is unnecessary. We have `.display-none` already.

Reply:
Since `.errors` has `display: flex`, which override `display:none` of `.display-none`, `.errors` and `.display-none` are in the same element. That's the reason why I added this.